### PR TITLE
Add torch 2.5.1 composer tests

### PR DESCRIPTION
**Pull request by j316chuck**
# What does this PR do?
Add torch 2.5.1 composer tests

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -262,7 +262,7 @@ RUN if [ -n "$MOFED_VERSION" ] ; then \
 RUN if [ -n "$CUDA_VERSION" ] ; then \
         pip${PYTHON_VERSION} install --upgrade --no-cache-dir ninja==1.11.1 && \
         pip${PYTHON_VERSION} install --upgrade --no-cache-dir --force-reinstall packaging==22.0 && \
-        MAX_JOBS=1 pip${PYTHON_VERSION} install --no-cache-dir --no-build-isolation flash-attn==2.6.3; \
+        pip${PYTHON_VERSION} install flash-attn==2.6.3; \
```
Comment (mvpatel2000) [inline docker/Dockerfile:265]: why these changes?

**Review by mvpatel2000 — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (j316chuck) [reply docker/Dockerfile:265]: just testing if we can not spend 2 hours building flash attention .

**Comment by dakinggg** [comment]
btw, FA has pre built wheels that we could probably use instead of pip instal: https://github.com/Dao-AILab/flash-attention/releases/tag/v2.7.0.post2 (see "assets" under each release)
